src/command_modules/azure-cli-botservice/azure/cli/command_modules/botservice/kudu_client.py

```diff
@@ -30,9 +31,9 @@ def __init__(self, cmd, resource_group_name, name, bot):
         self.bot_site_name = None
 
     def download_bot_zip(self, file_save_path, folder_path):
-        """Download bot's source code from KUDU.
+        """Download bot's source code from Kudu.
```

Comment:
This too.

Reply:
This method isn't exposed to the user, similar to `get_bot()` the docstring is for maintainable code.